nextflow.config

```diff
@@ -12,12 +12,25 @@ params {
     // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
-    mcl_inflation              = null
-    busco_lineages_path        = null
+    
+    // OrthoFinder MCL inflation parameters to test
+    mcl_inflation              = '1.0,2.0,3.0'
 
     // BUSCO options
-    lineage                    = null
-    config_file                = null
+    busco_lineages_path        = null
+
+    // Annotation options
+    download_annots = 'none'
+    
+    // Orthogroup filtering options
+    min_num_spp_per_og = 4
+    min_num_grp_per_og = 1
+    max_copy_num_spp_tree = 5
+    max_copy_num_gene_trees = 10
+    
+    // IQ-TREE options
+    tree_model = 'TEST'
```

Comment:
Asking this one last time so I am sure:
> What I mean is: if someone ran this with the default value of "TEST", will the pipeline fail or run in a semi-intelligent way? If not, we should either have a sensible default or not have a default and throw an error if not specified.

Reply:
Thought I posted a response that seems to be missing for whatever reason. 

This makes it generally useable, but perhaps slower if something more pointed is specified. Basically, this means it can be run with either nucleotide or amino acid sequences using defaults. But, I haven't tested the whole workflow with nucleotides, so maybe I should just specify something specific to amino acids that's sensible, and I can point to this in the readme, eventually building out nucleotide usage in later versions?

....I'm going to do that - I've convinced myself (the second time writing this response, lol).